src/concrete/ml/common/serialization/decoder.py

```diff
@@ -202,6 +203,9 @@ def object_hook(d: Any) -> Any:
                 model_class.__name__: model_class for model_class in serializable_classes
             }
 
+        if type_name == "Exactness":
```

Comment:
why do we store exactness objects in one of our object ?

Reply:
I don't know. I wonder if we test serialisation on our qmodules. I just dumped and loaded and got an error that exactness could not be serialised so I fixed it.

Reply:
yes we do test serialization for QMs ! But I don't get where we would store an `Exactness` instance in QMs ! 🤔  Doesn't look right

Reply:
I doubt we have QM tests or this would have been spotted.

Reply:
Alright so there was a probem with our QM serialisation tests indeed. We never test the case where we have rounding in the QM for serialisation. Fixed in https://github.com/zama-ai/concrete-ml/pull/778/commits/1cccb7c5307537501bd16a56f652a825c0ccf1e8